docs/toastandroid.md

```diff
@@ -67,58 +67,6 @@ const styles = StyleSheet.create({
 export default App;
 ```
 
-### Imperative hack
-
-The ToastAndroid API is imperative, but there is a way to expose a declarative component from it as in this example:
-
-```SnackPlayer name=Advanced%20Toast%20Android%20API%20Example&supportedPlatforms=android
-import React, {useState, useEffect} from 'react';
-import {View, StyleSheet, ToastAndroid, Button, StatusBar} from 'react-native';
-
-const Toast = ({visible, message}) => {
-  if (visible) {
-    ToastAndroid.showWithGravityAndOffset(
-      message,
-      ToastAndroid.LONG,
-      ToastAndroid.BOTTOM,
-      25,
-      50,
-    );
-    return null;
-  }
-  return null;
-};
```

Comment:
I removed this whole section because it is 100% not kosher for a render function to hav e side effects like this.